### PR TITLE
Supports customized JS/CSS assets

### PR DESCRIPTION
**Pull request by chenjiandongx**

### CODE REVIEW

File: charts/pie_test.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPieAssetsBeforeRender(t *testing.T) {
+	bar := NewBar()
+	assert.Equal(t, bar.JSAssets.Values, []string{"echarts.min.js"})
```
Comment (geange) [inline charts/pie_test.go:12]:
```suggestion
	assert.Equal(t, []string{"echarts.min.js"}, bar.JSAssets.Values)
```
[func Equal(t TestingT, expected, actual interface{}, msgAndArgs ...interface{}) bool](https://github.com/stretchr/testify/blob/92707c0b2d501c60de82176c4aa1cf880abac720/assert/assertions.go#L334)

File: charts/wordcloud_test.go [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+func TestWordCloudAssetsBeforeRender(t *testing.T) {
+	bar := NewBar()
+	assert.Equal(t, bar.JSAssets.Values, []string{"echarts.min.js"})
+}
+
+func TestWordCloudAssetsAfterRender(t *testing.T) {
+	bar := NewBar()
+	err := bar.Render(os.Stdout)
+	assert.NoError(t, err)
+	assert.Equal(t, bar.JSAssets.Values, []string{host + "echarts.min.js"})
```
Comment (geange) [inline charts/wordcloud_test.go:19]:
```suggestion
	assert.Equal(t, []string{host + "echarts.min.js"}, bar.JSAssets.Values)
```

File: charts/wordcloud_test.go [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestWordCloudAssetsBeforeRender(t *testing.T) {
+	bar := NewBar()
+	assert.Equal(t, bar.JSAssets.Values, []string{"echarts.min.js"})
```
Comment (geange) [inline charts/wordcloud_test.go:12]:
```suggestion
	assert.Equal(t, []string{"echarts.min.js"}, bar.JSAssets.Values)
```

File: opts/global.go [hunk 4]
```diff
@@ -896,22 +896,49 @@ type Colors []string
...
 	opt.JSAssets.Init("echarts.min.js")
 	opt.CSSAssets.Init()
+
+	opt.customizedJSAssets.Init()
+	opt.customizedCSSAssets.Init()
+}
+
+// AddCustomizedJSAssets adds the customized javascript assets which will not be added the `host` prefix.
+func (opt *Assets) AddCustomizedJSAssets(assets ...string) {
+	for i := 0; i < len(assets); i++ {
```
Comment (geange) [inline opts/global.go:915]: I prefer to use for-range personally🐶。(no need to judge every time)

Comment (chenjiandongx) [reply opts/global.go:915]: It works as expected and I think it's no need to follow the `individual-code-style`

**Comment by chenjiandongx** [comment]
@geange Polish code according to the review feedback

**Review by geange — APPROVED** [review]

**Review by geange — APPROVED** [review]

**Pull request merged**